### PR TITLE
Create Secluded.yml

### PR DESCRIPTION
**Pull request by ghost**
I hope my new project fits up-for-grabs policies. thanks!

### CODE REVIEW

File: _data/projects/Secluded.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: Secluded
+desc: Secluded is a webpage that tries to be isolated from web crawers although publically visible. It is a crawler behaviour experiment
```
Comment (ritwik12) [inline _data/projects/Secluded.yml]: Did you meant?
```suggestion
desc: Secluded is a webpage that tries to be isolated from web crawlers although publically visible. It is a crawler behaviour experiment
```
or is this intentional?

Comment (ritwik12) [reply _data/projects/Secluded.yml]: Please correct if this is a typo @bacloud14

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/Secluded.yml (hunk 1 above)
Comment (ghost) [reply _data/projects/Secluded.yml]: yes you are right @ritwik12 
thank you so much

Comment (ritwik12) [reply _data/projects/Secluded.yml]: Welcome :)

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@bacloud14 Thanks!!

**Pull request merged**